hooks/brc20/useBrc20TransferExecute.ts

```diff
@@ -0,0 +1,137 @@
+import { useCallback, useState } from 'react';
+
+import { NetworkType, UTXO } from 'types';
```

Comment:
hey @victorkirov for some reason this was fixed before and now was reverted. right?

Reply:
btw eslint or ts should complain about this right?

Reply:
Nope, that was from the first commit and hasn't changed since. What's the issue? The non-relative import?

Reply:
I just changed it to relative

Reply:
The way we've configured typescript for this project, it actually sees it as a valid import because we've set an alias:
<img width="206" alt="image" src="https://github.com/secretkeylabs/xverse-core/assets/3322667/17452c29-534e-4040-9fb2-c8a966727160">

I don't think we have anything at build time to process those aliases though, so not sure how it's supposed to work 🤔

Reply:
you're right. was another file that you fixed an import. i don't know why was causing errors on mobile if there is an alias

Reply:
Did the import path change fix it?